Build/Chakra.Build.Clang.Default.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Not officially supported! Configuration to build with clang on windows -->
-  <PropertyGroup Condition="'$(Clang)'=='ClangCL'">
+  <PropertyGroup Condition="'$(Clang)'=='ClangCL' And '$(ForceMSVC)'==''">
     <PlatformToolset>LLVM-vs2014</PlatformToolset>
```

Comment:
is vs2014 still correct?

Reply:
I believe that it is; the version packaged with VS matches this. There are GitHub repos with newer versions (llvm-vs2014 uses "clang with Microsoft CodeGen version 3.8.0", while there's GitHub repos for [5.0](https://github.com/arves100/llvm-vs2017-integration) and [6.0](https://github.com/arves100/llvm-vs2017-integration) third-party).